previews/aws-lambda-simple-queue-service.md

```diff
@@ -27,6 +27,10 @@ Since the initial cost of starting an `AwsLambdaSQSEndpoint` endpoint can be hig
 
 snippet: function-definition
 
+### Queue creation
+
+Transport installers are not supported. The required queues will have to be [provisioned manually](/transports/sqs/operations-scripting.md#create-resources).
```

Comment:
> The required queues will have to be [provisioned manually](/transports/sqs/operations-scripting.md#create-resources).

Maybe better as:

> The creation of the required queues can be scripted via the [CLI](/transports/sqs/operations-scripting.md#create-resources).

Reply:
Convered in https://github.com/Particular/docs.particular.net/pull/4991